File: Aula 09/Exemplo3B2.java
import java.util.Scanner;

public class Exemplo3B2 {
    public static void main(String[] args) {
        double[] salario = new double[5];
        int i, func;
        Scanner sc = new Scanner(System.in);
        for(i=0; i<2; i++){
            System.out.print("Digite o salário do " +(i+1) +"º funcionário: ");
            salario[i] = sc.nextDouble();
        }
        try {
            System.out.println("Digite o código do funcionário 0-4");
            func = sc.nextInt(i);
            System.out.println("Salário do 4º funcionário antes do método: " + salario[func]);
            calculaAumento(salario);
            System.out.println("Salário do 4º funcionário antes do método: " +salario[func]);
        } catch (Exception e) {
            //System.out.println(e);
            System.out.println("Funcionário não encontrado");
        }
    }
    public static void calculaAumento(double salario[]){
        for(int i=0; i< salario.length; i++){
            salario[i] = salario[i]*1.03;
        }
    }
}

File: Aula 09/Exemplo3B2.py
def calculaSalario(salario, i):
    salario[i] = salario[i] * 1.03
    
salario = []
for i in range(3):
    leia = float(input(f'Digite o salário do {i+1}º funcionário >_ '))
    salario.append(leia)

try:
    i = int(input('Qual o funcionário vc deseja verificar? '))
    print(f'Salário do {i+1}º funcionário antes do método: {salario[i]}')
    calculaSalario(salario, i)
    print(f'Salário do {i+1}º funcionário após o método: {salario[i]}')
except:
    print('Funcionário não encontrado')